app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -79,6 +80,9 @@ class EventsFragment : Fragment() {
         val lastVisible = linearLayoutManager.findLastVisibleItemPosition()
         val itemsChanged = lastVisible - firstVisible + 1 // + 1 because we start count items from 0
         val start = if (firstVisible - itemsChanged > 0) firstVisible - itemsChanged else 0
-        eventsRecyclerAdapter.notifyItemRangeChanged(start, itemsChanged + itemsChanged)
+        if (!eventsViewModel.loadEventsAgain)
```

Comment:
When will it ever get called

Reply:
The default value is false. So every time the app starts, the animation is seen or if the user clicks on Events in the bottom navigation bar ,the animation is seen.

Reply:
The boolean variable name is confusing. Please change to a proper name and comment wherever the vale is changed